app/views/katello/api/v2/subscriptions/base.json.rabl

```diff
@@ -18,7 +18,8 @@ attributes :virt_only
 
 node :host, :if => lambda { |sub| sub && sub.host } do |subscription|
   {
-    id: subscription.host.uuid,
+    id: subscription.host.host_id,
+    uuid: subscription.host.uuid,
```

Comment:
if we do have to include uuid here, i'm hesitant to  call it uuid as host has a uuid attribute which really doesn't get used very much (normally its nil).  If the uuid has to be used, could rename it to something like 'subscription_facet_uuid' ?